### PR TITLE
Update to emscripten 1.39.2 and fix example.

### PR DESCRIPTION
**Pull request by jplevyak**
Signed-off-by: John Plevyak <jplevyak@gmail.com>

### CODE REVIEW

File: api/wasm/cpp/README.md [hunk 1]
```diff
@@ -181,8 +181,8 @@ It is possible later versions will work, e.g.
 
 ```bash
 ./emsdk update-tags
-./emsdk install latest-upstream
-./emsdk activate latest-upstream
+./emsdk install latest
+./emsdk activate latest
 ```
 
 However 1.39.0-upstream is known to work.
```
Comment (PiotrSikora) [inline api/wasm/cpp/README.md]: Update this to `1.39.2`.

Comment (jplevyak) [reply api/wasm/cpp/README.md]: Done.

Comment (PiotrSikora) [reply api/wasm/cpp/README.md]: Not done... at least not pushed.

**Comment by jplevyak** [comment]
Got it now.  PTAL

**Pull request merged**